synadm/cli/user.py

```diff
@@ -500,6 +500,80 @@ def modify(ctx, helper, user_id, password, password_prompt, display_name,
         click.echo("Abort.")
 
 
+# for these placeholders, function similarly to the modify command, but in
+# it's own command.
+# TODO: find 3pid modify command
+
+
+@user.command()
+@click.argument("user_id", type=str)
+@click.argument("mxc_uri", type=str)
+@click.pass_obj
+def profile_picture(helper, user_id, mxc_uri):
+    """
+    Set a profile picture for a user by the MXC URI.
+    """
+    # TODO
+    pass
+
+
+@user.command()
+@click.argument("user_id", type=str)
+@click.argument("display_name", type=str)
+@click.pass_obj
+def display_name(helper, user_id, display_name):
+    """
+    Set the display name of a user.
+    """
+    # TODO
+    pass
+
+
+@user.command()
+@click.argument("user_id", type=str)
+@click.pass_obj
+def reactivate(helper, user_id):
+    # TODO
+    # TODO: does this require a password? merge into password? user
+    # experience?
+    pass
+
+
+@user.command()
+@click.argument("user_id", type=str)
+@click.pass_obj
+def lock(helper, user_id):
+    """
+    Lock a user account, preventing them from logging in or using the
+    account.
+    """
+    # TODO
+    pass
+
+
+@user.command()
+@click.argument("user_id", type=str)
+@click.pass_obj
+def unlock(helper, user_id):
+    """
+    Unlock a user account, allowing a locked account to log in and use the
+    account.
+    """
+    # TODO
+    # merge into lock? no because then it's hard to find
+    pass
+
+
+@user.command()
+@click.argument("user_id", type=str)
+@click.pass_obj
+def admin(helper, user_id):
+    # TODO
+    # removal of admin privileges is in here as well, which isn't consistent
+    # with lock and unlock, but what's the word for un-admin?
```

Comment:
We could go the "two word route" here as well:

Would be nicely sorted in `synadm user -h`:
`synadm user admin-grant`
`synadm user admin-revoke`

Would be kind of actual phrases:
`synadm user grant-admin`
`synadm user revoke-admin`